draft-ietf-quic-qpack.md

```diff
@@ -547,9 +559,9 @@ have outstanding (unacknowledged) references.
 An encoder MUST ensure that a header block which references a dynamic table
 entry is not received by the decoder after the referenced entry has already been
 evicted, and MUST ensure that the decoder will not suffer head-of-line blocking
-if the decoder has not opted to receive blocking references. Even if the decoder
-is willing to process blocking references, the encoder might choose to avoid
-them in certain cases.
+if the decoder has not opted to receive vulnerable references. Even if the
```

Comment:
End the sentence after "evicted".  Then "An encoder also respects the limit set by the decoder on the number of header blocks that are allow to be blocked."